src/integration-test/java/apptest/BusinessInformationIT.java

```diff
@@ -30,9 +31,8 @@ void test1_fetchingBusinesInformation() throws Exception {
 
 	@Test
 	void test2_fetchBusinessInformation_shouldOnlyExcludeMissingInformation() {
-		String partyId = "fb2f0290-3820-11ed-a261-0242ac120003";
 		setupCall()
-			.withServicePath("/information/" + partyId + "?organizationName=Some%20Organization")
+			.withServicePath("/information/" + PARTY_ID + "?organizationName=Some%20Organization")
```

Comment:
Could probably use a constant for "?organizationName=Some%20Organization" aswell.